ssg/build_yaml.py

```diff
@@ -1640,7 +1640,8 @@ def to_xml_element(self, env_yaml=None):
             check_content_ref.set("href", "oval-unlinked.xml")
             check_content_ref.set("name", self.id_)
 
-        if self.ocil or self.ocil_clause:
+        oval_content_only = self.oval_external_content is not None
+        if (self.ocil or self.ocil_clause) and not oval_content_only:
```

Comment:
Technically, other rules other than `security_patches_up_to_date` could have external content, and these rules would be allowed to have OCIL.
At the moment the only rule using `external_content` is `security_patches_up_to_date`, but this could be a problem if folks out there have "private" content that uses external content.